recipes/libvips/meta.yaml

```diff
@@ -0,0 +1,121 @@
+{% set name = "libvips" %}
+{% set version = "8.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libvips/libvips/releases/download/v{{ version }}/vips-{{ version }}.tar.gz
+  sha256: a0ee255a2a1ebfea5b2dff2a780824d7157a78c010d7ddd531279aacefbf2539 
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
```

Comment:
Inspire yourself from https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/meta.yaml#L49

to find where different dependencies should be included.

Reply:
Oh that's really helpful, thanks!